src/components/Detail/QueenDetail.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const QueenDetail = ({ name, image, winner, missCongeniality, quote }) => (
+  <section>
+    <h2>{name}</h2>
+    <img src={image} />
+    <p>Winner? {winner}</p>
+    <p>MissCongeniality? {missCongeniality}</p>
```

Comment:
It looks like the reason these two fields aren't populating is because both are coming in as booleans. One way you can solve it is by adding ternaries with string like this:
```
    <p>Winner? {winner ? 'Yes' : 'No'}</p>
    <p>MissCongeniality? {missCongeniality ? 'Yes' : 'No'}</p>
```